uktrade/data-hub-frontend
# Add banner alerting users to switch from IE to a modern browser

## Description of change
We are slowly deprecating the use of IE as soon it won't be available on the cirrus machines (EDGE will replace IE) so we are encouraging our users to use a modern browser for a better experience. To encourage users to stop using IE I have added a banner that will only show if you are on IE, this banner encourages the use of an alternative browser.

## Test instructions
- Open the app up in IE (use Browserstack if needed)
- Every route should have the banner showing at the top.

## Screenshot
![Screenshot 2021-02-19 at 10 58 54](https://user-images.githubusercontent.com/10154302/108705661-844f2e00-7505-11eb-9468-85043b13590e.png)


## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
